.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Describe a bug with single-spa-angular
+---
+
+### Demonstration
+
+<!--
+Please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem via
+https://stackblitz.com
```

Comment:
Can we do better than linking them to these tools, by providing a stackblitz / codesandbox template? I'm not sure I'd be able to get single-spa-angular working with code sandbox since it requires schematics changes, `ng add`, etc. And if I can't do it, I'm sure a lot of other people won't be able to do it either.

Reply:
_Link to github repository_ then?

Reply:
That could work. If stackblitz/codesandbox have support for running `ng add` commands, including schematics, we could create a template there, too.

Alternatively, if we just want to merge this in for now, we could recommend providing a github repo (without providing them a template) instead of linking to code sandbox / stackblitz which might not be easy for people to set up.